workbench-core/authorization/src/dynamicAuthorization/ddbDynamicAuthorizationPermissionsPlugin.test.ts

```diff
@@ -244,11 +234,20 @@ describe('DDB Dynamic Authorization Permissions Plugin tests', () => {
           Items: [mockIdentityPermissionItem]
         });
 
-      const { data } = await dynamoDBDynamicPermissionsPlugin.getIdentityPermissionsByIdentity({
+      const { data, paginationToken } =
+        await dynamoDBDynamicPermissionsPlugin.getIdentityPermissionsByIdentity({
+          identityId: sampleGroupId,
+          identityType: sampleGroupType
+        });
+      expect(data.identityPermissions).toStrictEqual([mockIdentityPermission]);
+      expect(paginationToken).toStrictEqual(base64PaginationToken);
```

Comment:
I'd recommend testing that the entire received response matches the expected response object rather than testing each field separately. This will help ensure we update tests if we add more fields to the response object

Reply:
Each field can be different per response while others can stay the same, testing an entire response can be a plus but I don't think it is necessary here